src/lib/components/workflow/filter-search/filter-list.svelte

```diff
@@ -134,7 +134,7 @@
     {/if}
   {/each}
   {#if hasMoreFilters}
```

Comment:
Also unrelated to this PR, but it looks like the `BooleanFilter` could use a `borderRadiusModifier`

![Screenshot 2023-08-29 at 4 04 03 PM](https://github.com/temporalio/ui/assets/15069288/dc4c767c-c3c0-418c-975f-21c90049f1bd)